test/scripts/generate-imported-targets-from-snyk.test.ts

```diff
@@ -212,7 +215,7 @@ describe('projectToTarget', () => {
   it('succeed to convert Github / Gitlab project name to target', async () => {
     const project = {
       name: 'lili-snyk/huge-monorepo:cockroach/build/builder/Dockerfile',
-      branch: 'main',
+      targetReference: 'main',
```

Comment:
this should not change, it should remain `branch`

Reply:
so you would like to convert the v3 output into the v1 output then?

Reply:
ignore that, just read the slack chat. I'll write a converter